code/modules/mob/living/simple_animal/hostile/statue.dm

```diff
@@ -174,16 +174,11 @@
 /mob/living/simple_animal/hostile/statue/say()
 	return 0
 
-//Turn to dust when gibbed
-/mob/living/simple_animal/hostile/statue/gib(var/animation = 0)
-	dust(animation)
```

Comment:
Technically removing this proc allows you to actually gib statues, instead of always having them turn to dust. Honestly I don't think that's particularly important though.